### PR TITLE
task/cw-0044 Looking for a benefactor

### PR DESCRIPTION
**Pull request by WhatWasThatt**

### CODE REVIEW

File: tasks/looking-for-a-benefactor/newAvg.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function newAvg(arr, newavg) {
```
Comment (askomar) [inline tasks/looking-for-a-benefactor/newAvg.js:1]: newavg - what style is it?

**Review by askomar — CHANGES_REQUESTED** [review]

**Review by askomar — DISMISSED** [review]
Maybe nice, but I recommend Misha to check

**Review by KarinaKlevzhits — COMMENTED** [review]
It's very good job!!Perfect!

**Review by KarinaKlevzhits — DISMISSED** [review]

**Review by mTeeeur — APPROVED** [review]
Looks good

**Review by Rascass — APPROVED** [review]
good

**Review by askomar — APPROVED** [review]
Nice

**Review by KarinaKlevzhits — APPROVED** [review]

**Pull request merged**